infra/params.dev.json

```diff
@@ -71,6 +71,12 @@
         "enableSearch": {
             "value": true
         },
+        "enableObservability": {
+            "value": true
+        },
+        "dataProductTeamEmail": {
+            "value": "email@domain.com"
```

Comment:
Please leave this blank. Otherwise, we will be sending lots of emails to this email address.